doc/manual/functions.rst

```diff
@@ -677,6 +677,24 @@ will be fused with other nested "dot" calls.  ``X .+= Y`` is equivalent
 to ``X .= X .+ Y`` and will eventually result in a fused in-place assignment.
 Similarly for ``.*=`` etcetera.)
 
+Note that there is a difference between ``max.(a,b)``, which ``broadcast``s :func:`max`
```

Comment:
I think you need a space (maybe backslash-escaped?) between the trailing ``` `` ``` and the s